src/main/java/teammates/ui/automated/PrepareEmailWorkerAction.java

```diff
@@ -0,0 +1,163 @@
+package teammates.ui.automated;
+
+import java.util.List;
+
+import teammates.common.datatransfer.CommentSendingState;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.exception.TeammatesException;
+import teammates.common.util.Assumption;
+import teammates.common.util.EmailType;
+import teammates.common.util.EmailWrapper;
+import teammates.common.util.HttpRequestHelper;
+import teammates.common.util.Const.ParamsNames;
+import teammates.logic.core.CommentsLogic;
+import teammates.logic.core.EmailGenerator;
+import teammates.logic.core.EmailSender;
+import teammates.logic.core.FeedbackResponseCommentsLogic;
+import teammates.logic.core.FeedbackSessionsLogic;
+
+/**
+ * Task queue worker action: prepares large number of emails to be sent via task queue.
+ */
+public class PrepareEmailWorkerAction extends AutomatedAction {
```

Comment:
Just to confirm, the only logging activities removed are related to the admin activity log, right?